APISpecification.yaml

```diff
@@ -0,0 +1,100 @@
+openapi: 3.0.3
+info:
+  title: Ensembl Resolver API
+  description: Resolver API resolves external urls to Ensembl
+  contact:
+    email: ensembl-webteam@ebi.ac.uk
+  version: 0.0.1
+servers:
+  - url: https://resolver.ensembl.org
+tags:
+  - name: resolver
+    description: Resolver API resolves external urls to Ensembl
+paths:
+  /resolve/{gene_stable_id}:
+    get:
+      summary: Resolve stable ID with optional query params
+      description: Resolves to a beta url when a stable id wih optional query params provided
+      parameters:
+        - name: gene_stable_id
+          in: path
+          description: Gene stable id used to represent a gene in a genome.
+          required: true
+          explode: true
+          schema:
+            type: string
+            example: ENSG00000221914.11
+        - name: type
+          in: query
+          description: Defines a type - gene/transcript/variant etc
+          required: false
+          schema:
+            type: string
+            enum: [gene, transcript, variant]
```

Comment:
To support using this service as an API there should be an end point that provides these enums.